src/main/java/io/vertx/ext/mongo/DownloadOptions.java

```diff
@@ -0,0 +1,64 @@
+package io.vertx.ext.mongo;
```

Comment:
I wonder if DownloadOptions is too generic and this needs to be named more specifically, e.g. That it's retrieval via GridFS

Reply:
Changed to GridFsDownloadOptions